src/libraries/Common/src/System/Net/Http/Http3/QPack/H3StaticTable.cs

```diff
@@ -0,0 +1,215 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.Collections.Generic;
+using System.Net.Http;
+using System.Text;
+
+#if KESTREL
+namespace Microsoft.AspNetCore.Server.Kestrel.Core.Internal.Http3.QPack
+#else
+namespace System.Net.Http.QPack
+#endif
+{
+    internal class H3StaticTable
+    {
+        private static readonly H3StaticTable _instance = new H3StaticTable();
```

Comment:
s_instance

Reply:
This whole class needs to be made static. Going to leave a comment and do post-merge.